testing/trino-benchto-benchmarks/src/test/java/io/trino/sql/planner/AbstractTpcdsCostBasedPlanTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.sql.planner;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.plugin.tpcds.TpcdsMetadata;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+
+public abstract class AbstractTpcdsCostBasedPlanTest
```

Comment:
I would avoid the additional level in inheritance hierarchy for `AbstractTpcdsCostBasedPlanTest` and `AbstractTpchCostBasedPlanTest`, it's not really saving us much in terms of code repetition.